compiler/il/symbol/OMRSymbol_inlines.hpp

```diff
@@ -213,19 +213,6 @@ OMR::Symbol::isPinningArrayPointer()
    return self()->isAuto() && _flags.testAny(PinningArrayPointer);
    }
 
-void
```

Comment:
There is a small typo "propgate" in the commit message. If you can fix that up and at the same time mention this cleanup of the below code in the commit message, that would be great.